source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
Maybe it's okay to use real `global` instead of `fakeWindow` which is exported from dom? I understand that it's global namespace pollution, but at least it's consistent with the browser behaviour. If we need to fix that then it's need to be done for browser too.